fix-frontend: carrito-responsive

rosita2423 opened the pull request:
El flujo del carrito fue mejorado para que fuera responsive además de que se agreago la pantalla de resumen de compra.

rosita2423 committed:
fix-frontend: carrito-responsive

El flujo del carrito fue mejorado para que fuera responsive además de que se agreago la pantalla de resumen de compra

ARTIC000 commented on apps/frontend_web/src/app/Carrito/page.tsx:
```diff
@@ -7,7 +8,7 @@ import { useProducts } from '@/hooks/useProduct'
 import { useCart } from '@/hooks/useCart'
 import { ResumenCompra } from '@/components/resumenCompra'
 import { CarrouselProducts } from '@/components/CarouselProducts'
-import BotonRegresar  from "@/components/BotonRegresar";
```
Esto se debe de dejar tal cual como esta

rosita2423 committed:
fix-frontend: Importación Botones Regresar

Por un error de vercel se cambió la manera en que se importaba el componente botón regresar.
